### DISCUSSION TITLE
Is there a way to backup your images?

### DISCUSSION THREAD

**Discussion by zygoballus** [Symbiota/community-board#581] [Q&A] [answered]
I know you can backup all your data by using the "Download Backup Data File" link in the Administration Control Panel, but is there any way to backup your images?

**Comment by themerekat** [comment, marked as answer]
@zygoballus , no, not through the Symbiota portal, but you can batch download from the DwC-A them using this R script: https://zenodo.org/records/7922418

**Reply by zygoballus** [reply]
Thanks! That is very helpful.
